plugin.video.focussat/resources/settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
```

Comment:
it'd be nice if you could move this to the new style settings, it's about time we start to think about deprecating the older alternative